tests/datasets/test_datasets.py

```diff
@@ -44,4 +44,25 @@ def test_demo_dataset():
         "offline_data_method": "not a real method",
     })
     with pytest.raises(NotImplementedError):
-        create_dataset(env)
+        create_dataset(env, 123)
+
+
+def test_demo_replay_dataset():
+    """Test demo+replay dataset creation with Covers env.
+    """
+    utils.update_config({
+        "env": "cover",
+        "approach": "trivial_learning",
+        "offline_data_method": "demo+replay",
+        "offline_data_planning_timeout": 500,
+        "offline_data_num_replays": 3,
+    })
+    env = CoverEnv()
+    dataset = create_dataset(env, 123)
+    assert len(dataset) == 5 + 3
+    assert len(dataset[-1]) == 2
+    assert len(dataset[-1][0]) == 2
+    assert len(dataset[-1][1]) == 1
+    for _, actions in dataset:
+        for action in actions:
+            assert action.has_option()
```

Comment:
Add this test too:
```
# Test that data does not contain options since approach is random
utils.update_config({
    "env": "cover",
    "approach": "random",
})
env = CoverEnv()
dataset = create_dataset(env, 123)
assert len(dataset) == 5 + 3
assert len(dataset[-1]) == 2
assert len(dataset[-1][0]) == 2
assert len(dataset[-1][1]) == 1
for _, actions in dataset:
    for action in actions:
        assert not action.has_option()
```